src/control/server/drpc.go

```diff
@@ -27,15 +27,69 @@ import (
 	"context"
 	"os"
 	"path/filepath"
+	"time"
 
 	"github.com/golang/protobuf/proto"
 	"github.com/pkg/errors"
 
+	mgmtpb "github.com/daos-stack/daos/src/control/common/proto/mgmt"
 	"github.com/daos-stack/daos/src/control/drpc"
 	"github.com/daos-stack/daos/src/control/logging"
 	"github.com/daos-stack/daos/src/control/security"
 )
 
+const (
+	defaultRetryAfter = 250 * time.Millisecond
+)
+
+type daosStatusResp struct {
+	Status int32 `protobuf:"varint,1,opt,name=status,proto3" json:"status,omitempty"`
+}
+
+func (dsr *daosStatusResp) String() string {
+	return ""
+}
```

Comment:
Out of curiosity, why don't we want to print anything here?

Reply:
I had to add a String() method to satisfy the proto.Message interface. daosStatusResp is just used to extract the status (down in `makeDrpcCall()`) without needing to know the real response type.